.ci.yaml

```diff
@@ -548,15 +548,15 @@ targets:
 
   - name: Linux flutter_packaging_test
     recipe: packaging/packaging
-    timeout: 60
```

Comment:
what's the default? i thought 60 WAS the default.

Reply:
It's 30 minutes. Ideally, we would have a tool that gets historical data to send PRs to change this, but I'm not sure how to get the BigQuery data from Dart. 

https://cs.opensource.google/flutter/cocoon/+/main:app_dart/lib/src/model/proto/internal/scheduler.proto;l=40